llvm/tools/llvm-exegesis/lib/SerialSnippetGenerator.cpp

```diff
@@ -53,6 +53,8 @@ computeAliasingInstructions(const LLVMState &State, const Instruction *Instr,
     if (OtherOpcode == Instr->Description.getOpcode())
       continue;
     const Instruction &OtherInstr = State.getIC().getInstr(OtherOpcode);
+    if (ET.getIgnoredOpcodeReasonOrNull(State, OtherInstr.getOpcode()))
```

Comment:
For some reason this filter was removed when RISCV's exegesis was first introduced. But it really should be here.